Article/Article.less

```diff
@@ -37,14 +38,31 @@
     color: grey;
     cursor: pointer;
     transform: translate(-50%);
-    background-color: white;
+    background-color: #C2B97F;
   }
 }
-
-.article-open {
-  height: 400px;
+.article--open {
+  animation-name: expand;
+  animation-duration: 5s;
+  animation-fill-mode: forwards;
 }
 
+.article--close {
+  animation-name: close;
+  animation-duration: 5s;
+  animation-fill-mode: forwards;
+}
 
+@keyframes expand{
```

Comment:
This was a really professional looking animation. It was just subtle enough to make your articles events look that much better, but it was hard to point to the reason why.